### ISSUE TITLE
[MD030/MD036] Seemingly inconsistent behavior

### ISSUE THREAD

**Issue by borisroman** [rvben/rumdl#278]
Hi,

Given the following config;

```toml
[MD004]
style = "dash"

[MD049]
style = "asterisk"

[MD050]
style = "asterisk"
```

Here are some markdown examples and the warnings.

`[MD036] Emphasis used instead of a heading: 'This text is extremely important'`
```markdown
**This text is extremely important**
```

`No issue`
```markdown
**This text is *extremely* important**
```

`[MD036] Emphasis used instead of a heading: 'This text is very important'`
```markdown
*This text is very important*
```

`[MD030] Spaces after list markers (Expected: 1; Actual: 0)`
```markdown
*This text is **very** important*
```

For consistency I would either assume both example 2 and 4 to produce a `[MD036] Emphasis used instead of a heading` issue. Or as an alternative have example 4 produce no issue.

I added the config for MD004 as reference because it might influence the MD030 detection.

**Comment by rvben** [comment]
Thanks for raising this issue, @borisroman!

This was fixed in v0.0.210. The issue was that MD030 was incorrectly flagging ordered lists when the content started with certain patterns. This has been resolved.

**Comment by borisroman** [comment]
Thank you for fixing!
